src/Commands/ExportPostmanCommand.php

```diff
@@ -140,6 +140,11 @@ public function handle(): void
 
     protected function getReflectionMethod(array $routeAction): ?object
     {
+        // Hydrates the closure if it is an instance of Opis\Closure\SerializableClosure
+        if (is_string($routeAction['uses']) && Str::startsWith($routeAction['uses'], 'C:32:"Opis\\Closure\\SerializableClosure') !== false) {
```

Comment:
@almightynassar I would execute `unserialize` to make sure it's serialized before doing this.

Reply:
So are you suggesting and extra clause to the if statement? Such as:

```
if (is_string($routeAction['uses']) && Str::startsWith($routeAction['uses'], 'C:32:"Opis\\Closure\\SerializableClosure') !== false && ($hydrated = unserialize($routeAction['uses'])) {
    $routeAction['uses'] = $hydrated->getClosure();
}
```

For reference I used the same process used by the Laravel framework (see [RouteAction.php](https://github.com/laravel/framework/blob/50f24dddf9d83becd3b0967f88bbf12fb550ca99/src/Illuminate/Routing/RouteAction.php#L104))

Reply:
Because it's defined in `RouteAction` and this package relies on `illuminate\routing`, we could import the `RouteAction` class here and change this to:

```
if (\Illuminate\Routing\RouteAction::containsSerializedClosure($routeAction) {
    $routeAction['uses'] = hydrated->getClosure();
}
```

This way we can (hopefully) keep in sync with any changes to the serialization of Laravel routes in the core routing package.

Reply:
@almightynassar I like the second option of calling the static method.

Reply:
@tomirons just made the change